commit 855873f6ae8f8d0f398ab952423a50f144e557a6
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2019-06-11T17:01:29+00:00 (4 months ago)
    
    MANTA-4039 binder load balancer never met a packet it didn't queue

diff --git a/deps/mname-balancer b/deps/mname-balancer
index e17d6c4..1f9bbdb 160000
--- a/deps/mname-balancer
+++ b/deps/mname-balancer
@@ -1 +1 @@
-Subproject commit e17d6c493141abbededee49371adddcf65887804
+Subproject commit 1f9bbdbab411aecce21d4ec4f9b8862b199521d1
